pkg/apis/options/cookie.go

```diff
@@ -17,6 +17,7 @@ type Cookie struct {
 	Secure   bool          `flag:"cookie-secure" cfg:"cookie_secure"`
 	HTTPOnly bool          `flag:"cookie-httponly" cfg:"cookie_httponly"`
 	SameSite string        `flag:"cookie-samesite" cfg:"cookie_samesite"`
+	Minimal  bool          `flag:"cookie-minimal" cfg:"cookie_minimal"`
```

Comment:
Should this be part of the session options rather than the cookie options?

Reply:
That was actually the first place I looked, then I remembered the CookieOptions struct was removed in a recent PR 😄 

You are right, it likely makes much more sense there.

What do you think the naming convention of cookie session store based flags should be? `cookie` prefix is for HTTP cookie options (with Secret a little overloaded for session encryption). So that is out as a parallel to `redis` prefix for redis session flags.

Reply:
I guess `session-cookie-minimal` (which will also be the struct path kinda) would work? WDYT? Or `minimal-session-cookie`, and just make a note that it only affects cookie based sessions?